packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail.tsx

```diff
@@ -160,6 +161,7 @@ const RancherDetail = ({
                   color={ODS_THEME_COLOR_INTENT.primary}
                   size={ODS_BUTTON_SIZE.sm}
                   variant={ODS_BUTTON_VARIANT.stroked}
+                  target={'_blank' as any}
```

Comment:
You have to `import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';` and use `OdsHTMLAnchorElementTarget._blank`